Assignment2/rentalCarCost.js

```diff
@@ -0,0 +1,4 @@
+function rentalCarCost(d) {
+  // Your solution here
+  return d >= 7? 40*d - 50 : d >=3 ? 40*d - 20 : 40*d

```

Comment:
this is okay, but if, else if, else is more readable. but its fine for now.

Reply:
Ok I will do changes.